utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/AppSecIast.java

```diff
@@ -274,6 +277,16 @@ String secureXPath(final ServletRequest request) {
         return "XPath secure";
     }
 
+    @PostMapping("/xss/test_insecure")
+    String insecureXSS(final ServletRequest request, final ServletResponse response) throws IOException {
+        return xssExamples.insecureXSS(response.getWriter(), request.getParameter("param"));
```

Comment:
It's very weird that you return a string from your endpoint and use the writer of the response, your method should be void.